bundle/src/main/java/com/cognifide/cq/cqsm/core/scripts/ScriptImpl.java

```diff
@@ -130,4 +127,9 @@ public Date getLastModified() {
 	public String getData() {
 		return scriptContent.getData();
 	}
+
+	@Override
+	public Date getDryRunLast() {

```

Comment:
`getLastDryRun` sounds a little bit better

Reply:
Yeah, but current name is valid with name's convention in cqsm_nodetypes